Sources/PackageSyntax/Formatting.swift

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
Some of this file could be folded into SwiftSyntax's new SwiftBasicFormat module, although there are some assumptions here which are more likely to hold in a manifest compared to arbitrary swift sources